### ISSUE TITLE
Custom user schemes could be blockers for startup wizard

### ISSUE THREAD

**Issue by Defman21** [Komodo/KomodoEdit#1706]
### Short Summary

http://forum.komodoide.com/t/unable-to-start-komodo-edit-x-with-linux-debian/2878 in this post, the OP reproduced a very strange problem. Some of their schemes are corrupted (not a valid ksf file or there's a non-ASCII character) and Startup Wizard could not load itself because of it.

### Platform Information

*Komodo Edit or IDE?* Edit/IDE (no matter)
*Komodo Version?* 10.0.0
*Operating System (and version)?* no matter

### Additional Information
```
[2016-06-15 12:40:31,168] [ERROR] koScintillaSchemeService: Syntax Error loading scheme /home/dabluemel/.komodoedit/10.0/schemes/base16-brewer-dark.ksf:
Traceback (most recent call last):
  File "/home/dabluemel/Komodo-Edit-10/lib/mozilla/components/koScintillaSchemeService.py", line 121, in _execfile
    execfile(fname, namespace)
  File "/home/dabluemel/.komodoedit/10.0/schemes/base16-brewer-dark.ksf", line 2
SyntaxError: Non-ASCII character '\xc3' in file /home/dabluemel/.komodoedit/10.0/schemes/base16-brewer-dark.ksf on line 2, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
[2016-06-15 12:40:31,169] [ERROR] SchemeBase: Unable to load Komodo color scheme: {'userDefined': 1, 'filepath': u'/home/dabluemel/.komodoedit/10.0/schemes/base16-brewer-dark.ksf'}
=== OR ===
[2016-06-15 10:16:53,326] [ERROR] koScintillaSchemeService: Syntax Error loading scheme /home/dabluemel/.komodoedit/10.0/schemes/Meins2.ksf:
Traceback (most recent call last):
  File "/home/dabluemel/Komodo-Edit-10/lib/mozilla/components/koScintillaSchemeService.py", line 121, in _execfile
    execfile(fname, namespace)
  File "/home/dabluemel/.komodoedit/10.0/schemes/Meins2.ksf", line 1
    version 39
             ^
SyntaxError: invalid syntax
[2016-06-15 10:16:53,327] [ERROR] SchemeBase: Unable to load Komodo color scheme: {'userDefined': 1, 'filepath': u'/home/dabluemel/.komodoedit/10.0/schemes/Meins2.ksf'}

[2016-06-15 12:40:31,176] [ERROR] console-logger: TypeError: s is null (2) in chrome://komodo/content/startupWizard/startupWizard.js:214
Traceback (most recent call last):
  File "chrome://komodo/content/startupWizard/startupWizard.js", line 214, in 

[2016-06-15 12:40:31,178] [ERROR] console-logger: TypeError: w is null (2) in chrome://komodo/content/bindings/windows.xml:319
Traceback (most recent call last):
  File "chrome://komodo/content/bindings/windows.xml", line 319, in

**Comment by Naatan** [comment]
This is already fixed in 10.0.1, which is due soon.

**Comment by tonyattwood** [comment]
Hi,


I'd like to  invite you  to  my party, it's gonna be a surprise for my family, here  is the invitation <http://expert.icss-canada.net/e4mibbu>


Sent from my iPhone, tony.attwood@ntlworld.com
